docker-compose.yaml

```diff
@@ -128,11 +128,11 @@ services:
       options:
         max-size: "20m"
     build:
-      context: ./
-      dockerfile: Dockerfile.emulations
+      context: ./scripts
```

Comment:
Let's put the dockerfile full path to the context

Reply:
We can't add it to context, as we need to access the common folder.